app/utils/validation.ts

```diff
@@ -53,6 +53,16 @@ export const timeIsAfter = (otherField, message) => (value, context) => {
   return [true];
 };
 
+export const isValidAllergy =
+  (
+    message = 'La feltet stå tomt hvis du ikke har noen allergier/preferanser'
+  ) =>
+  (value: string) => {
+    const notValidAnswers = ['ingen', 'ingenting', 'nei', 'nope', 'nada'];
+
+    return [!notValidAnswers.includes(value.toLowerCase()), message];
```

Comment:
Could you explain how this syntax works! It looks interesting :)

Reply:
Not sure which part you're referring to. The validator needs an array with two values; a boolean and a message. The boolean indicates if the validation has passed or not, and the message is what is given to the user. What we're actually creating here is a function returning a function which returns the array.